.ci/.jenkins_exclude.yml

```diff
@@ -13,6 +13,15 @@ exclude:
   - RUBY_VERSION: docker.elastic.co/observability-ci/jruby:9.1-7-jdk
     FRAMEWORK: rails-4.2
 
+  - RUBY_VERSION: ruby:2.4
+    FRAMEWORK: rails-6.0
+  - RUBY_VERSION: ruby:2.3
+    FRAMEWORK: rails-6.0
+  - RUBY_VERSION: jruby:9.1
+    FRAMEWORK: rails-6.0
+  - RUBY_VERSION: docker.elastic.co/observability-ci/jruby:9.1-7-jdk
+    FRAMEWORK: rails-6.0-rc1
```

Comment:
@mikker I was looking at this PR to see what had to be done to test Rails 6 and noticed this line still has `rc1`. Is that intentional?

Reply:
Nope! Good catch.